src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -464,6 +463,81 @@ gl_sframe activity_classifier::predict_per_window(gl_sframe data,
   return result;
 }
 
+gl_sframe activity_classifier::classify(gl_sframe data,
+                                        std::string output_frequency) {
+  if (output_frequency.empty()) {
+    output_frequency = "per_row";
+  } else if (output_frequency != "per_row" &&
+             output_frequency != "per_window") {
+    log_and_throw(output_frequency +
+                  " is not a valid option for output_frequency.  "
+                  "Expected one of 'per_row' or 'per_window'.");
+  }
+
+  // perform inference
+  std::unique_ptr<data_iterator> data_it =
+      create_iterator(data, false, false, false);
+  gl_sframe raw_preds_per_window = perform_inference(data_it.get());
+
+  // lambda function for getting the max probability for each prediction
+  auto max_prob = [=](const flexible_type& ft) {
```

Comment:
I don't think this lambda captures anything. Let's use `[]` instead of `[=]` to make that explicit